WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/history/ScanHistoryListFragment.kt

```diff
@@ -30,15 +39,30 @@ class ScanHistoryListFragment : ViewPagerFragment(R.layout.scan_history_list_fra
     }
 
     private fun initAdapter() {
+        recycler_view.adapter = ScanAdapter(imageManager, uiHelpers)
+        recycler_view.itemAnimator = null
     }
 
     private fun initViewModel(site: SiteModel) {
         viewModel = ViewModelProvider(this, viewModelFactory).get(ScanHistoryListViewModel::class.java)
+        parentViewModel = ViewModelProvider(parentFragment as ViewModelStoreOwner, viewModelFactory).get(
+                ScanHistoryViewModel::class.java
+        )
+        viewModel.start(site, parentViewModel)
         setupObservers()
-        viewModel.start(site)
     }
 
     private fun setupObservers() {
+        viewModel.uiState.observe(
+                viewLifecycleOwner,
+                { listItems ->
+                    refreshContentScreen(listItems)
+                }
+        )
```

Comment:
Minor (🔍): It can fit in one line.

```suggestion
        viewModel.uiState.observe(viewLifecycleOwner, { refreshContentScreen(it) })
```